distribution/docs/src/main/resources/content/_configuring/banner-markings-content-extractor.adoc

```diff
@@ -0,0 +1,88 @@
+:title: Banner Markings Extractor
+:type: configuringConfigFile
+:status: published
+:summary: Text file banner markings.
+:order: 132
+
+The Banner Markings Extractor parses the text of PDFs, Microsoft Office documents, plain text files,
+ or any other file from which the <<_tika_input_transformer,TikaTransformer>> can extract text. The
+ ${branding} default behavior is to apply high-water security settings when ingesting a document.
+ This is sufficient for documents with no security markings, but is incorrect when working with
+ classified documents that include security mark-up in the text.
+
+The marking extractors parses a banner marking and attempts to create new attributes on the
```

Comment:
`extractor parses` or `extractors parse`?

Reply:
This is a tougher question than it first appears. Inside the bundle there are two marking extractors and they extract different things. In retrospect, I think this is something I'd like to leave as an implementation detail, so I will use the singular instead of the plural.